ui/pages/confirmations/hooks/useTypedSignSignatureInfo.ts

```diff
@@ -0,0 +1,46 @@
+import { useMemo } from 'react';
+
+import {
+  isOrderSignatureRequest,
+  isPermitSignatureRequest,
+  isSignatureTransactionType,
+} from '../utils';
+import { SignatureRequestType } from '../types/confirm';
+import { parseTypedDataMessage } from '../../../../shared/modules/transaction.utils';
+import { TokenStandard } from '../../../../shared/constants/transaction';
+import { MESSAGE_TYPE } from '../../../../shared/constants/app';
+import { TypedSignSignaturePrimaryTypes } from '../constants';
+
+export const useTypedSignSignatureInfo = (
+  confirmation: SignatureRequestType,
+) => {
+  if (!confirmation) {
+    return {};
+  }
+  if (
```

Comment:
Minor, could this be combined with the above condition so we have a single guard with all applicable conditions?

Reply:
yep done